### PR TITLE
Basic assignment slide

### PR DESCRIPTION
**Pull request by FinnStutzenstein**

### CODE REVIEW

**Comment by MaximilianKrambach** [comment]
Additional data I will need for a displayed poll:
- [ ]  assignment title
- [ ] poll.published (boolean)
- [ ]   poll options (candidates)
  - [ ] names only if poll is not published. 
  - [ ] yes/no/abstain (or votes) values
  - [ ] a 100% base if 100% base is activated (the client has calculation methods in assignment poll service, can these be used?)
  - [ ] valid, invalid, cast votes (if present)

File: client/src/app/slides/assignments/poll/poll-slide-data.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+export interface PollSlideData {
+    title: string;
+    assignments_poll_100_percent_base: string; // TODO: The type is somewhere defined, isn't it?
```
Comment (MaximilianKrambach) [inline client/src/app/slides/assignments/poll/poll-slide-data.ts]: `assignment-poll.service:AssignmentPercentBase` (currently not exported)

File: client/src/app/slides/assignments/poll/poll-slide-data.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+    poll: {
+        published: boolean;
+        description?: string;
+        has_votes?: boolean;
+        pollmethod?: string; // TODO: The type is somewhere defined, isn't it?
```
Comment (MaximilianKrambach) [inline client/src/app/slides/assignments/poll/poll-slide-data.ts]: `assignment-poll.service:AssignmentPollMethod`

File: client/src/app/slides/assignments/poll/poll-slide-data.ts [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+        votesno?: string; // TODO: same conversion needed as for the PollModel
```
Comment (MaximilianKrambach) [inline client/src/app/slides/assignments/poll/poll-slide-data.ts]: Are these (no, abstain on the per-poll level) even used anywhere?

Comment (FinnStutzenstein) [reply client/src/app/slides/assignments/poll/poll-slide-data.ts]: I do not know exactly. With the voting plugin they were showed. I think, we should show them, if they have any value.

**Comment by FinnStutzenstein** [comment]
@MaximilianKrambach The poll slide is done.

File: client/src/app/slides/assignments/poll/poll-slide-data.ts (hunk 2 above)
Comment (MaximilianKrambach) [reply client/src/app/slides/assignments/poll/poll-slide-data.ts]: I did this in my PR

**Comment by FinnStutzenstein** [comment]
@MaximilianKrambach Can this be merged, if you work on another branch/PR?

**Comment by MaximilianKrambach** [comment]
> @MaximilianKrambach Can this be merged, if you work on another branch/PR?

Yes @FinnStutzenstein

**Pull request merged**